replace hybrid with Apigee X/Hybrid

rockspore opened the pull request:
close #215

theganyo reviewed (dismissed):
LGMT. Are there any references to GCP or GCP Experience we would need to address?

rockspore commented:
> LGMT. Are there any references to GCP or GCP Experience we would need to address?

Good point! Replaced one exposed message and changed some comments from referring to GCP exp to referring to Apigee X/Hybrid.
